src/predefinedComponents/DetailsHeader/DetailsHeader.tsx

```diff
@@ -79,14 +79,28 @@ class DetailsHeader extends React.Component<DetailsHeaderProps, State> {
       <View style={[styles.headerWrapper, { backgroundColor }]}>
         <View style={styles.headerMenu}>
           <TouchableOpacity hitSlop={sizes.hitSlop} onPress={leftTopIconOnPress}>
-            {leftTopIcon && <Image style={styles.icon} resizeMode="contain" source={leftTopIcon} />}
+            {typeof leftTopIcon === 'function' ? (
```

Comment:
This seems like something which we could move to some kind of helper file as function, so this code would be easier to read